packages/application-components/src/components/dialogs/info-dialog/info-dialog.story.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, select, boolean } from '@storybook/addon-knobs';
+import withReadme from 'storybook-readme/with-readme';
+import { PORTALS_CONTAINER_ID } from '@commercetools-frontend/constants';
+import {
+  SecondaryButton,
+  Spacings,
+  Text,
+} from '@commercetools-frontend/ui-kit';
+import Readme from './README.md';
+import InfoDialog from './info-dialog';
+
+const DialogController = props => {
+  const [isOpen, toggle] = React.useState(false);
```

Comment:
I wouldn't do this even though I like it. That means anybody building an app with our application-components needs to be on the latest React version. Not that it's unlikely but it's a bit of a harsh restriction balancing the benefit of it (mainly being funky) 😄

Reply:
This is just the story for storybook, I don't think it's a problem to be honest.

Reply:
True. Missed that. All good then. But it is worthwhile being on the lookout 😁

Reply:
Our first hook component 🥺